app/views/applications/edit.html.erb

```diff
@@ -32,6 +32,7 @@
     <%= f.input :battlenet, label: "Battle.net ID" %>
     <%= f.input :logs, hint: "World of Logs, and/or Warcraft Logs" %>
     <%= f.input :computer, label: "Computer Specs", input_html: {rows: 5} %>
+    <%= f.input :ui, label: "UI Screenshot", input_html: {rows: 5} %>
```

Comment:
`:ui` is not the correct attribute name, should be the name of the field in the database, which you have not added yet.